<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="5" passed="4">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="50" started-at="2021-08-30T09:38:11Z" finished-at="2021-08-30T09:38:11Z">
    <groups>
      <group name="@smoke">
        <method signature="SampleTest1.tc0()[pri:0, instance:org.sample.test.SampleTest1@6f7fd0e6]" name="tc0" class="org.sample.test.SampleTest1"/>
        <method signature="SampleTest1.tc2()[pri:0, instance:org.sample.test.SampleTest1@6f7fd0e6]" name="tc2" class="org.sample.test.SampleTest1"/>
      </group> <!-- @smoke -->
      <group name="@Retest">
        <method signature="SampleTest1.tc4()[pri:0, instance:org.sample.test.SampleTest1@6f7fd0e6]" name="tc4" class="org.sample.test.SampleTest1"/>
      </group> <!-- @Retest -->
      <group name="@E2E">
        <method signature="SampleTest1.tc3()[pri:0, instance:org.sample.test.SampleTest1@6f7fd0e6]" name="tc3" class="org.sample.test.SampleTest1"/>
      </group> <!-- @E2E -->
      <group name="@sanity">
        <method signature="SampleTest1.tc1()[pri:0, instance:org.sample.test.SampleTest1@6f7fd0e6]" name="tc1" class="org.sample.test.SampleTest1"/>
      </group> <!-- @sanity -->
    </groups>
    <test name="Default test" duration-ms="50" started-at="2021-08-30T09:38:11Z" finished-at="2021-08-30T09:38:11Z">
      <class name="org.sample.test.SampleTest1">
        <test-method status="FAIL" signature="tc0()[pri:0, instance:org.sample.test.SampleTest1@6f7fd0e6]" name="tc0" duration-ms="11" started-at="2021-08-30T09:38:11Z" finished-at="2021-08-30T09:38:11Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertTrue(Assert.java:44)
at org.testng.Assert.assertTrue(Assert.java:54)
at org.sample.test.SampleTest1.tc0(SampleTest1.java:16)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tc0 -->
        <test-method status="PASS" signature="tc3()[pri:0, instance:org.sample.test.SampleTest1@6f7fd0e6]" name="tc3" duration-ms="1" started-at="2021-08-30T09:38:11Z" finished-at="2021-08-30T09:38:11Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tc3 -->
        <test-method status="PASS" signature="tc4()[pri:0, instance:org.sample.test.SampleTest1@6f7fd0e6]" name="tc4" duration-ms="2" started-at="2021-08-30T09:38:11Z" finished-at="2021-08-30T09:38:11Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tc4 -->
        <test-method status="PASS" signature="tc2()[pri:0, instance:org.sample.test.SampleTest1@6f7fd0e6]" name="tc2" duration-ms="1" started-at="2021-08-30T09:38:11Z" finished-at="2021-08-30T09:38:11Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tc2 -->
        <test-method status="PASS" signature="tc1()[pri:0, instance:org.sample.test.SampleTest1@6f7fd0e6]" name="tc1" duration-ms="0" started-at="2021-08-30T09:38:11Z" depends-on-methods="org.sample.test.SampleTest1.tc0" finished-at="2021-08-30T09:38:11Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tc1 -->
      </class> <!-- org.sample.test.SampleTest1 -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
